src/helpers/common.ts

```diff
@@ -30,6 +30,17 @@ export function parseJsonFromIpfs(jsonUri: string): Wrapped<JSONValue> | null {
     return null;
   }
 
+  if (ipfsHash[0] != 'Q' || ipfsHash[1] != 'm') {
+    log.info('INVALID IPFS HASH FOUND FOR URI {}', [jsonUri]);
+    return null;
+  }
+
+  if (ipfsHash == 'QmcfMBWdXMubqGGS6nYKNJXAeB9PTJpCH8NTuzN5uGLkvp' || ipfsHash == 'QmagSLXM2wp8sXqaAm9fVJvcCXq4JJormeWo9JtkBnFFjB'
```

Comment:
@robsecord This shouldn't be our long-term fix but this will fix the subgraph on Kovan for now as it manually catches the 3 IPFS hashes that currently don't resolve and cause the subgraph to get stuck and basically just skips them.